tests/Unit/Functions/rocketCleanMinify.php

```diff
@@ -0,0 +1,57 @@
+<?php
+namespace WP_Rocket\Tests\Unit\Functions;
+
+use WP_Rocket\Tests\Unit\FilesystemTestCase;
+use Brain\Monkey\Functions;
+
+/**
+ * @covers rocket_clean_minify()
+ * @group Functions
+ * @group Files
+ */
+class Test_RocketCleanMinify extends FilesystemTestCase {
+	protected $rootVirtualDir = 'cache';
```

Comment:
@Tabrisrp `$rootVirtualDir` is set to 'cache' in the base VirtualFilesystemTestCase here https://github.com/wp-media/phpunit/blob/master/Unit/VirtualFilesystemTestCase.php#L15